pom.xml

```diff
@@ -186,7 +185,6 @@
         <geronimoSpecsServletApiVersion>1.0</geronimoSpecsServletApiVersion>
         <greenmailVersion>1.5.0</greenmailVersion>
         <groovyVersion>2.4.4</groovyVersion>
-        <gsonVersion>2.5</gsonVersion>
         <guavaVersion>23.5-jre</guavaVersion>
         <hamcrestVersion>1.3</hamcrestVersion>
         <jacksonVersion>1.9.11</jacksonVersion>
```

Comment:
Is this used?

Reply:
Yes, it's on `./mule-integration-tests/integration/pom.xml` and the value is inherited from `./mule/pom.xml` through `./mule-integration-tests/pom.xml`